File: Basic/Loop/219432C.c
#include <stdio.h>
#define intll long long

void solution() {

	int n;
	scanf("%d", &n);

	int even = 0, odd = 0, pos = 0, neg = 0;
	for (int i = 1; i <= n; i++) {
		int x;
		scanf("%d", &x);

		// odd-even
		if (x % 2 == 0) {
			even++;
		} else {
			odd++;
		}

		// positive-negative
		if (x > 0) {
			pos++;
		} else if (x < 0) {
			neg++;
		}
	}

	printf("Even: %d\n", even);
	printf("Odd: %d\n", odd);
	printf("Positive: %d\n", pos);
	printf("Negative: %d\n", neg);
}

int main() {
	// char TEST_CASE = !1;
	int t = 1;
	// if (TEST_CASE == 1) {
	// 	scanf("%d", &t);
	// }

	while (t--) {
		solution();
	}
	return 0;
}

File: Basic/Loop/219432C.go
package main

import (
	"fmt"
)

func solution() {

	var n int
	fmt.Scanf("%d", &n)

	even, odd, pos, neg := 0, 0, 0, 0
	for i := 1; i <= n; i++ {
		var x int
		fmt.Scan(&x)

		// odd-even
		if x%2 == 0 {
			even++
		} else {
			odd++
		}

		// positive-negative
		if x > 0 {
			pos++
		} else if x < 0 {
			neg++
		}
	}

	fmt.Printf("Even: %d\n", even)
	fmt.Printf("Odd: %d\n", odd)
	fmt.Printf("Positive: %d\n", pos)
	fmt.Printf("Negative: %d\n", neg)
}

func main() {
	TEST_CASE := !true
	t := 1
	if TEST_CASE {
		fmt.Scanf("%d", &t)
	}

	for t != 0 {
		solution()
		t--
	}
}
